src/ProjectNeon/Assets/Features/Effects/Attack.cs

```diff
@@ -1,18 +1,42 @@
-﻿using System.Collections;
-using System.Collections.Generic;
+﻿
+
+
 using UnityEngine;
-using UnityEngine.Events;
 
-public sealed class Attack 
+public sealed class Attack  : Effect
 {
-    public Member Attacker { get; }
-    public Member Target { get; }
+    private Member _attacker;
+    public Member Attacker() { return this._attacker; }
```

Comment:
This should still be a property. 

You can make it mutable by having a private setter.

```
public Member Attacker { get; private set; }
```